src/topp/FeatureFinderMultiplex.cpp

```diff
@@ -259,20 +258,24 @@ class TOPPFeatureFinderMultiplex :
     // write feature map, consensus maps and blacklist
     if (!(out_.empty()))
     {
-      writeFeatureMap_(out_, algorithm.getFeatureMap());
+      FeatureMap& feature_map = algorithm.getFeatureMap();
+      feature_map.setPrimaryMSRunPath({in_}, exp);
+      writeFeatureMap_(out_, feature_map);
     }
     if (!(out_multiplets_.empty()))
     {
-      writeConsensusMap_(out_multiplets_, algorithm.getConsensusMap());
+      ConsensusMap& consensus_map = algorithm.getConsensusMap();
+      consensus_map.setPrimaryMSRunPath({in_}, exp);
```

Comment:
@lars20070 here you need to set the filename for every column in the consensus map. e.g., for standard silac you would set {in, in} for triple silac {in, in, in}.

Reply:
Ok. See latest changes.